build_tools/github_actions/configure_ci.py

```diff
@@ -264,7 +279,8 @@ def get_benchmark_presets(is_pr: bool, trailers: Mapping[str, str],
       preset_options = preset_options.union(
           option.strip() for option in trailer.split(","))
     preset_options = sorted(preset_options)
-    print(f"Using benchmark preset '{preset_options}' from trailers and labels")
+
+  print(f"Using benchmark preset '{preset_options}'")
```

Comment:
Prints are pretty harmless in this script, right? Might help future debugging to also print something when the presets aren't selected based on trailers and labels (in the `if not is_pr` and `elif is_llvm_integrate_pr` branches)

Reply:
SG. Done